.gitignore

```diff
@@ -7,3 +7,4 @@ __pycache__
 *.pyo
 *.swp
 .pytest_cache/
+venv/
```

Comment:
I think, this safe.
A newline would be nice.

Reply:
Sure thing.